conf/distro/nilrt.conf

```diff
@@ -25,3 +25,23 @@ KERNEL_IMAGEDEST = "boot/runmode"
 ROOT_HOME = "/home/admin"
 
 IMAGE_LINGUAS ?= "en-us en-us.iso-8859-1"
+
+BUILD_IMAGES_FROM_FEEDS = "1"
+
+# Get NIOE packages from local deploy feed and NI packages from web feed
+IPK_NIOE_SUBFEED_URI ?= "file://${DEPLOY_DIR_IPK}"
```

Comment:
I think these lines would be better moved to `nilrt.inc`. In theory, the difference between *there* and *here* is that `nilrt.inc` is _how_ to build the distro, and `nilrt.conf` is _what_ to put in the distro. In reality, we don't follow that separation very well.